wiki/Beatmap_ranking_procedure/Ranking_queue/id.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - qualified queue
+  - ranked queue
+  - ranking ETA
+  - antrian peringkat
+  - tahap kualifikasi
+---
+
+# Antrian peringkat beatmap
+
+**Antrian peringkat** (*Ranking queue*) mengatur pemindahan [beatmap-beatmap](/wiki/Beatmap) dari [Qualified](/wiki/Beatmap/Category#qualified) ke [Ranked](/wiki/Beatmap/Category#ranked). Setiap hari, 10 beatmap dari masing-masing [mode permainan](/wiki/Game_mode) dipindahkan dari Qualified ke Ranked, jika beatmap-beatmap tersebut sudah berada di kategori Qualified setidaknya selama 7 hari. Waktu di mana beatmap-beatmap tersebut menjadi ranked adalah acak.
```

Comment:
I'm curious why it's "beatmap-beatmap" -- honest mistake or a proper term?

Reply:
it's a way to represent the plural (s) (to some extent) in indonesian, by doubling the object next to it

beatmaps = beatmap-beatmap
mods= mod-mod
modes = mode-mode

something like that \o/

Reply:
wow!